site/docs/master/about.md

```diff
@@ -31,7 +31,7 @@ The **restore** operation allows you to restore all of the objects and persisten
 
 The default name of a restore is `<BACKUP NAME>-<TIMESTAMP>`, where `<TIMESTAMP>` is formatted as *YYYYMMDDhhmmss*. You can also specify a custom name. A restored object also includes a label with key `velero.io/restore-name` and value `<RESTORE NAME>`.
 
-You can also run the Velero server in restore-only mode, which disables backup, schedule, and garbage collection functionality during disaster recovery.
+You can also configure backup storage locations in read-only mode, which disables backup creation and deletion for the storage location during disaster recovery. 
```

Comment:
To be extra clear, I'd say: You can configure... in read-write and read-only modes. In the read-only mode...

I'd also `s/disaster recovery/restore` at the end of this sentence.

Reply:
addressed.